conf/rusk.toml

```diff
@@ -15,7 +15,7 @@ kadcast_id = 0x20
 public_address = '127.0.0.1:9000'
 listen_address = '127.0.0.1:9000' 
```

Comment:
Replace this with 'N/A' so that users are going to get an error if their detect_ip.sh is not working